project/accounts/views.py

```diff
@@ -188,6 +188,21 @@ def get(self, request, username=None):
         )
 
 
+class UserFollowing(LoginRequiredMixin, View):
+    """A view that shows list of users following user with given username"""
+
+    def get(self, request, username=None):
+        profile = get_object_or_404(Profile, user__username=username)
+
+        return TemplateResponse(
+            request,
+            "profile_followers.html",
```

Comment:
This template name should be consistent with the view and URL definition, which both use "profile following" and not "followers."